README.md

```diff
@@ -2,22 +2,29 @@
 
 Docker image to load user profiles into CCD.
 
+## Building
+
+Any commit or merge into master will automatically trigger an Azure ACR task. This task has been manually
+created using `/bin/deploy-acr-task.sh`. The task is defined in `acr-build-task.yaml`. Note: you will need a GitHub personal token defined in `GITHUB_TOKEN` environment variable to run deploy script (https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line).
```

Comment:
```suggestion
created using `./bin/deploy-acr-task.sh`. The task is defined in `acr-build-task.yaml`. Note: you will need a GitHub personal token defined in `GITHUB_TOKEN` environment variable to run deploy script (https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line).
```

What is that GITHUB_TOKEN for?
Should it be a service account?

Reply:
Yep webhook 👍 

It should be a service account! But running this task should be part of some automation too probably 🙈 

Ill update with more info